src/css/header.css

```diff
@@ -11,7 +11,7 @@ header {
 	justify-content: space-between;
 }
 
-a {
+header a {
```

Comment:
@RoelN You might have accidentally global selected `a` as i.s.o scoping it to header, which is why footer also received paddings on `a`'s.

Reply:
Good catch! This was definitley by accident, thanks for the fix 👍